vnext/Microsoft.ReactNative/IReactContext.h

```diff
@@ -57,8 +58,8 @@ struct ReactContext : winrt::implements<ReactContext, IReactContext> {
       JSValueArgWriter const &paramsArgWriter) noexcept;
 
  public: // IReactContext
-  // Not part of the public ABI interface
-  // Internal accessor for within the Microsoft.ReactNative dll to allow calling into internal methods
+         // Not part of the public ABI interface
+         // Internal accessor for within the Microsoft.ReactNative dll to allow calling into internal methods
```

Comment:
nit

Reply:
It looks like `yarn format` does this to these lines. Probably not worth adding suppression comments.